pkg/ads/ads.go

```diff
@@ -14,154 +14,146 @@
  * limitations under the License.
  */
 
-package ads
+ package ads
 
-import (
-	"fmt"
-	"log"
-	"net"
-	"strings"
-
-	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials/insecure"
-	"google.golang.org/protobuf/proto"
-	"google.golang.org/protobuf/types/known/anypb"
-	mcp "istio.io/api/mcp/v1alpha1"
-	networking "istio.io/api/networking/v1alpha3"
-	"istio.io/istio/pkg/adsc"
-	"istio.io/istio/pkg/config/schema/collections"
-	"istio.io/istio/pkg/model"
-	"kmesh.net/kmesh-coredns-plugin/pkg/dns"
-	"kmesh.net/kmesh-coredns-plugin/pkg/options"
-)
-
-type AdsController struct {
-	server    *dns.KmeshDNSServer
-	adsClient *adsc.ADSC
-	vip       string
-}
-
-func NewAdsController(dnsServer *dns.KmeshDNSServer) (*AdsController, error) {
-	c := &AdsController{
-		server: dnsServer,
-		vip:    options.GetConfig().VIP,
-	}
-	client, err := adsc.New(options.GetConfig().XDSAddress, &adsc.ADSConfig{
-		InitialDiscoveryRequests: configInitialRequests(),
-		ResponseHandler:          c,
-		Config: adsc.Config{
-			Workload:  options.GetConfig().ServiceNode,
-			Namespace: options.GetConfig().ServiceNameSpace,
-			Meta: model.NodeMetadata{
-				Generator: "api",
```

Comment:
why remove it?

Reply:
I thought your comment before said this is not necessary